### ISSUE TITLE
Why did you separate the theme into modules?

### ISSUE THREAD

**Issue by okps123** [LekoArts/gatsby-themes#152]
First of all, thank you for creating a nice theme.
I'm making my blog using gatsby-theme-minimal-blog.

However, the theme is split into modules, so changing the blog title or hero content requires modifying the code inside node_modules.

Therefore, the code that is being modified cannot be managed by git, and people who use it may find it difficult.

I wonder what you think about this.
Thank you.

**Comment by LekoArts** [comment]
Hi!

> the theme is split into modules

This is leveraging [Gatsby Themes](https://www.gatsbyjs.org/docs/themes/). It is a way of composing Gatsby sites together. It also allows you to [shadow files](https://www.gatsbyjs.org/docs/themes/shadowing/).

> so changing the blog title

The title at the top is set by the [`siteTitle`](https://github.com/LekoArts/gatsby-themes/blob/3db5f8374e8bea79d3929560a51d0f714bd00ae5/themes/gatsby-theme-minimal-blog/src/components/header.tsx#L29) defined in gatsby-config. Optionally you can also shadow the component.

> hero content

I've got a short paragraph on which file you need to shadow: https://github.com/LekoArts/gatsby-themes/tree/master/themes/gatsby-theme-minimal-blog#changing-the-hero-text

---

Bottom line is: You don't need to change content in node_modules :)
Does that clear things up?

**Comment by okps123** [comment]
Missing Shadowing.
Thank you.
